src/TUTORIAL.md

```diff
@@ -1115,9 +1115,15 @@ You should also take note that in this case, we don't need to call the `txCtx.re
 
 ### Backend Field Value Calculation
 
-When we work with our _Account_ records we have to provide the `passwordDigest` value to the API, which means the client code has to deal with the digest calculation, while the password check will be ultimately performed in the backend. That means that we will have to maintain the digest algorithms in sync on both the client and the server side. Besides, cyptographic digest functionality may not be readily available on the client side. All that means that we'd rather have the client send the account password in plain text when it creates and updates _Account_ records and have our web-service calculate the digests.
+When we work with our _Account_ records, we have to provide a `passwordDigest` value to the API, so code running on the client must compute the digest. However, ultimately the password check will be performed by the server. For this to work, we must maintain the digest algorithms on both the client and server, keeping them in sync. Also, another potential issue with this approach is, cryptographic digest functionality may not be readily available on the client.
 
-First, let's see how it can be done for the new account creation call. The record template submitted with the `POST` call to our `/accounts` endpoint can have fields that are not described in the record type definition. Such fields are simply ignored by the framework. So, we could include `password` field in the record template and have a hook in our `accounts.js` handler convert it to the `passwordDigest` value. That must happen before the template is validated (`passwordDigest` is a required field) and the hook for that is called `prepareCreateSpec`:
+Therefore, when the client creates and updates _Account_ records, we'd rather have it send the account password in plain text and have our web-service calculate the digest.
+
+**Important: You must be able to ensure clients may only send passwords in plain text, if they are using a secure encrypted connection with the server.**
+
```

Comment:
This is important only for live environments, so, yes, I think it's good to mention it, but let's something like "Make sure that in production environment sensitive information such as plain text passwords are exchanged between the client and the web-service over a secure connection (SSL)."

Reply:
I think it is an important note which should be distinguished from other notes.